### PR TITLE
fix(budget-create-page): apply datasource currency & apply v2 api

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Change client version from v1 to v2.
- Apply dataSource currency to budgetFormAmountPlanLastMonthsCost
image

### CODE REVIEW

File: apps/web/src/services/cost-explorer/budget/shared/budget-form/BudgetForm.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@ const createBudget = async () => {
 
     state.loading = true;
     try {
-        await SpaceConnector.client.costAnalysis.budget.create({
+        await SpaceConnector.clientV2.costAnalysis.budget.create({
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/budget/shared/budget-form/BudgetForm.vue:42]: praise: !!!!!!

File: apps/web/src/services/cost-explorer/budget/shared/budget-form/budget-form-amount-plan/BudgetFormAmountPlanLastMonthsCost.vue [hunk 2]
```diff
@@ -58,70 +73,80 @@ const state = reactive({
...
+        state.last3MonthsBudgets = results;
     } catch (e) {
         ErrorHandler.handleError(e);
         state.last3MonthsBudgets = [];
     }
 };
 
-watch(() => state.budgetListParams, () => {
+watch([() => props.projectId, () => props.dataSourceId, () => props.projectGroupId], () => {
+    console.log('showList', state.showList);
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/budget/shared/budget-form/budget-form-amount-plan/BudgetFormAmountPlanLastMonthsCost.vue:142]: nitpick: console.log here!

**Review by piggggggggy — APPROVED** [review]
LGTM~!

**Review by yuda110 — APPROVED** [review]
LGTM!

**Pull request merged**